### PR TITLE
add scala 2.11 & 2.12 compile job in CI

### PR DESCRIPTION
**Pull request by gadomsky**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -68,6 +68,8 @@ jobs:
       script: ./ciRunSbt.sh clean ui/slow:test
     - name: "Management tests"
       script: ./ciRunSbt.sh clean management/it:test
+    - name: "Scala 2.11-2.12 compile"
+      script: CROSS_BUILD=true ./ciRunSbt.sh clean compile
```
Comment (mproch) [inline .travis.yml]: test:compile?
Can we make it run only if "original" CROSS_BUILD==false?

Comment (arkadius) [reply .travis.yml]: I think that @mproch want to have both `compile:compile` and `test:compile`. Am I right?

File: .travis.yml [hunk 2]
```diff
@@ -68,6 +68,9 @@ jobs:
       script: ./ciRunSbt.sh clean ui/slow:test
     - name: "Management tests"
       script: ./ciRunSbt.sh clean management/it:test
+    - name: "Scala 2.11-2.12 compile"
```
Comment (arkadius) [inline .travis.yml]: WDYT about changing name of this task to `Cross scala compile`. Let's keep information about all supported versions in one place, in sbt...

Comment (mproch) [reply .travis.yml]: :+1:

File: .travis.yml (hunk 1 above)
Comment (gadomsky) [reply .travis.yml]: fixed

File: .travis.yml (hunk 2 above)
Comment (gadomsky) [reply .travis.yml]: fixed

**Pull request merged**